utils/logalloc.cc

```diff
@@ -2540,7 +2540,7 @@ region_group::execution_permitted() noexcept {
 }
 
 future<>
-region_group::start_releaser(scheduling_group deferred_work_sg) {
+region_group::start_releaser(scheduling_group deferred_work_sg) noexcept {
```

Comment:
It's probably outside of the scope of this change, but why does this function start with yield?

Reply:
e40fb438f5470234194848b6cb69439b4afe453f